### DISCUSSION TITLE
Comment Emoji's Dropdown (website)

### DISCUSSION THREAD

**Discussion by PhoenixAndStuf** [ScratchAddons/ScratchAddons#977] [Ideas]
_(Important note: I posted this in the issues tab but it didnt seem to get much traction, so hopefully the idea can get some love here)_

Back at it again with a new idea!

**Describe the new addon.**

When your commenting and want to show an emoji, you would usually have to manually type in _ meow _, _ B) _, _ 10mil _, so on and so forth.

But this addon would add a little dropdown menu with buttons labeled "Cat" for _ meow _, "Confetti" for _ 10mil _, ect.

These labels would also have a the emoji display itself on the label. The labels, when pressed, would add the text, such as _ 10mil _, for you instead of manually typing it in.

**Resolves:**

Nothing

**Reasons to add it:**

Just a handy little quality of life feature.

**Comment by retronbv** [comment]
Nice, I think people would benefit from this

**Comment by Marc92020** [comment]
This would be useful so I don't need to look at the scratch wiki to see how to type the emojis in.

**Reply by PhoenixAndStuf** [reply]
> This would be useful so I don't need to look at the scratch wiki to see how to type the emojis in.

Yep!

**Reply by gohoski** [reply]
Yeah, that is gonna be pretty helpful.

**Comment by GrahamSH-LLK** [comment]
> (Important note: I posted this in the issues tab but it didnt seem to get much traction, so hopefully the idea can get some love here)

Please don't.

**Reply by PhoenixAndStuf** [reply]
ok, sorry this topic is old anyway

**Reply by cobaltt7** [reply]
yeah this should be closed

**Reply by GrahamSH-LLK** [reply]
It's a discussion, you can't close it.

**Reply by Marc92020** [reply]
then it should be answered

**Comment by cobaltt7** [comment]
Oh ok

Paul Reid

________________________________
